components/colorarea/index.css

```diff
@@ -1,68 +1,107 @@
-%spectrum-ColorControl-hiddenField {
-  opacity: 0;
-  position: absolute;
-  top: 0; left: 0;
-  width: 100%; height: 100%;
-  z-index: 0;
-  margin: 0;
-  pointer-events: none;
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+.spectrum-ColorArea {
+  --spectrum-colorarea-border-radius: var(--spectrum-color-area-border-rounding);
+  --spetrcum-colorarea-border-color: rgba(0, 0, 0, 0.1); /* TODO replace with token --spectrum-color-area-border-color and --spectrum-color-area-border-opacity using RGBA function */
+  --spectrum-colorarea-disabled-background-color: var(--spectrum-disabled-background-color);
+  --spectrum-colorarea-border-width: var(--spectrum-color-area-border-width);
+  --spectrum-colorarea-height: var(--spectrum-color-area-height);
+  --spectrum-colorarea-width: var(--spectrum-color-area-width);
+  --spectrum-colorarea-minimum-width: var(--spectrum-color-area-minimum-width);
+  --spectrum-colorarea-minimum-height: var(--spectrum-color-area-minimum-height);
+  --spectrum-colorhandle-size: 16px; /* TODO replace with new token for ColorHandle, placeholder for old token var(--spectrum-colorhandle-size) */
 }
 
-%spectrum-ColorControl-handle--focused {
-  /* Bigger handle when focused */
-  width: calc(var(--spectrum-colorhandle-size) * 2);
-  height: calc(var(--spectrum-colorhandle-size) * 2);
-
-  margin-left: calc(-1 * var(--spectrum-colorhandle-size));
-  margin-top: calc(-1 * var(--spectrum-colorhandle-size));
+/* Windows High Contrast Mode */
+@media (forced-colors: active) {
+  .spectrum-ColorArea {
+    --highcontrast-colorarea-border-color-disabled: GrayText;
+    --highcontrast-colorarea-border-color: Canvas;
```

Comment:
I am using a highcontrast property for the border color so that the border is not displayed in WHCM. This is to match how ColorWheel appears in WHCM, which displays without a border color in WHCM because it uses a background-color instead of a border and cannot use a border color. Since these components are often displayed together this helps with consistency. Open to feedback or suggestions on other ways to handle this.